pom.xml

```diff
@@ -839,8 +839,8 @@
     <httpclient.version>4.5.13</httpclient.version>
     <httpcore.version>4.4.13</httpcore.version>
     <metrics-core.version>3.2.6</metrics-core.version>
-    <jackson.version>2.14.1</jackson.version>
-    <jackson.databind.version>2.14.1</jackson.databind.version>
+    <jackson.version>2.15.2</jackson.version>
```

Comment:
Latest jackson is 2.15.3; just synced with hbase-thirdparty here. Please let me know if should make this latest here!